test/conversations.spec.js

```diff
@@ -0,0 +1,126 @@
+const describe = require('mocha').describe;
+const expect = require('chai').expect;
+const path = require('path');
+const client = require('twilio')('ACxxx', 'fake_token');
+const conversations = require('../src/conversations/conversations');
+
+ApiMocker = require('./api_mocker').ApiMocker;
+
+const mocker = new ApiMocker(path.join(__dirname, 'fixtures'));
+
+describe('conversations', function() {
+  describe('on create', function() {
+    it('makes a request and returns a ConversationInstance', async function() {
+      client.httpClient.request = mocker.mock([{ resource: 'Conversations' }]);
```

Comment:
Here we can use stubs or mocks from `sinon` to fake the Twilio client. You can take a look at this [test case](https://github.com/TwilioDevEd/sample-pay-service/blob/master/test/services/payment-details.spec.js) for an example on how to work with sinon. In that case I used spies, but you can create mocks too.